doc/doc_03_cap_01.rst

```diff
@@ -0,0 +1,328 @@
+Paradigmi per la progettazione delle interfacce di servizio
+==============================================================
+
+Una trattazione completa dei paradigmi per la progettazione e
+realizzazione delle interfacce di servizio esula dagli scopi del
+presente documento. Il lettore interessato ad approfondire gli aspetti
+metodologici può riferirsi a [1]_, [2]_ . La breve discussione che viene
+presentata in questa sezione, condotta anche attraverso esempi, vuole
+ricordare al lettore la problematica dell'\ *impedance mismatch* tra
```

Comment:
Questo concetto viene comunemente utilizzato per la mappatura OO-RDBMS, legati a elementi teorici (strutture a grafo nell'OO ed algebra relazionale nei RDBMS). Utilizzarlo in altri ambiti può essere una forzatura...